qgreenland/config/project.py

```diff
@@ -1,14 +1,15 @@
 from qgreenland.config.constants import PROJECT_CRS
+from qgreenland.constants import ASSETS_DIR
 from qgreenland.models.config.project import ConfigProject
 
 project = ConfigProject(
     crs=PROJECT_CRS,
     boundaries={
         'background': {
-            'filepath': '{assets_dir}/latitude_shape_40_degrees.geojson',  # noqa: FS003
+            'filepath': f'{ASSETS_DIR}/latitude_shape_40_degrees.geojson',  # noqa: FS003
```

Comment:
This broke the tests because when we evaluate `ASSETS_DIR` we get an absolute path. Depending on the computer that evaluates, that absolute path could be anything. The next PR will fix this.